recipe/meta.yaml

```
@@ -12,9 +12,10 @@ source:
     # https://github.com/Azure/azure-sdk-for-cpp/issues/4784
     # https://github.com/Azure/azure-sdk-for-cpp/pull/4785
     - 0001-remove-wil-from-exported-config.patch
+    - 0001-Complete-RTTI-fix-for-macOS.patch  # [osx]
 
 build:
-  number: 0
+  number: 1
```

I let you apply the suggestion if it LGTY.

```suggestion
  number: 2
```